com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/UsageDataSession.java

```diff
@@ -114,6 +115,7 @@ public void recordResponse(Response response) {
                 requestEventMap.remove(response.request_seq);
             }
             long duration = responseMillis - requestMillis;
+            commandPerfCountMap.compute(command, (k, v) -> (v == null ? 0 : v.intValue()) + (int) duration);
```

Comment:
It stores total duration of a certain command. Will average duration be more useful?